doc/src/manual/mathematical-operations.md

```diff
@@ -348,16 +359,18 @@ Julia applies the following order of operations, from highest precedence to lowe
 | Control flow   | `&&` followed by `\|\|` followed by `?`                                                           |
 | Assignments    | `= += -= *= /= //= \= ^= ÷= %= \|= &= ⊻= <<= >>= >>>=`                                            |
 
-### Elementary Functions
+For a complete list of *every* Julia operator's precedence, see the top of this file:
+https://github.com/JuliaLang/julia/blob/master/src/julia-parser.scm
```

Comment:
Oops! Sorry about that. Thankfully, the emails aren't deleted. 😄

Here's what I had last said, and i was waiting for input:

> Do you have a suggestion? Maybe just something like this?:
> ```
>  [julia/src/julia-parser.scm](https://github.com/JuliaLang/julia/blob/master/src/julia-parser.scm)
> ```


I'll do that for now and see what you think! :)